### ISSUE TITLE
Change name of column and row in the tooltip

### ISSUE THREAD

**Issue by Nicolabo** [rstudio/d3heatmap#71]
Is it possible to customize name of column and row in the tooltip, to provide different labels that are placed on the y-axis and x-axis?

**Comment by talgalili** [comment]
Hi @Nicolabo 

The heatmaply R package is a new implementation of interactive cluster heatmaps in R which relies on the plotly R package. This is based on a fork of my work in d3heatmap.

The package is availabale from CRAN:
https://cran.r-project.org/package=heatmaply

The feature you've requested is now available by using the following code:
```r
library(heatmaply)
heatmaply(mtcars, label_names = c("x_name", "y_name", "some_value"))
```

![image](https://cloud.githubusercontent.com/assets/976006/25064547/fa663e24-2204-11e7-9f66-447a3582017e.png)

**Comment by aaronxs** [comment]
Hi Tal,

I tried running the sample using mtcars you gave but I get an error  
``Error in `[.data.table`(dt, , c(.I, NA), by = keyVars) : object '.I' not found``

Any idea what caused this? Thanks!
Aaron

**Comment by talgalili** [comment]
Why not use heatmaply for such things?

**Comment by aaronxs** [comment]
Yes I used heatmaply and got that error message, I copied the code you have if your response above. Sorry I should have been more clear.
